packages/dev/serializers/src/glTF/2.0/glTFMaterialExporter.ts

```diff
@@ -84,240 +65,89 @@ function getFileExtensionFromMimeType(mimeType: ImageMimeType): string {
-     * @param exportMaterials
-     * @param mimeType texture mime type
-     * @param hasTextureCoords specifies if texture coordinates are present on the material
-     * @returns promise that resolves after all materials have been converted
-     */
-    public _convertMaterialsToGLTFAsync(exportMaterials: Set<Material>, mimeType: ImageMimeType, hasTextureCoords: boolean) {
-        const promises: Promise<IMaterial>[] = [];
-        exportMaterials.forEach((material) => {
-            if (material.getClassName() === "StandardMaterial") {
-                promises.push(this._convertStandardMaterialAsync(material as StandardMaterial, mimeType, hasTextureCoords));
-            } else if (material.getClassName().indexOf("PBR") !== -1) {
-                promises.push(this._convertPBRMaterialAsync(material as PBRMaterial, mimeType, hasTextureCoords));
-            } else {
-                Tools.Warn(`Unsupported material type: ${material.name}`);
-            }
-        });
-
-        return Promise.all(promises).then(() => {
-            /* do nothing */
-        });
-    }
+    const a = dielectricSpecular.r;
+    const b = (diffuse * oneMinusSpecularStrength) / (1.0 - dielectricSpecular.r) + specular - 2.0 * dielectricSpecular.r;
+    const c = dielectricSpecular.r - specular;
+    const D = b * b - 4.0 * a * c;
+    return Scalar.Clamp((-b + Math.sqrt(D)) / (2.0 * a), 0, 1);
+}
 
-    /**
-     * Makes a copy of the glTF material without the texture parameters
-     * @param originalMaterial original glTF material
-     * @returns glTF material without texture parameters
-     */
-    public _stripTexturesFromMaterial(originalMaterial: IMaterial): IMaterial {
-        const newMaterial: IMaterial = {};
-        if (originalMaterial) {
-            newMaterial.name = originalMaterial.name;
-            newMaterial.doubleSided = originalMaterial.doubleSided;
-            newMaterial.alphaMode = originalMaterial.alphaMode;
-            newMaterial.alphaCutoff = originalMaterial.alphaCutoff;
-            newMaterial.emissiveFactor = originalMaterial.emissiveFactor;
-            const originalPBRMetallicRoughness = originalMaterial.pbrMetallicRoughness;
-            if (originalPBRMetallicRoughness) {
-                newMaterial.pbrMetallicRoughness = {};
-                newMaterial.pbrMetallicRoughness.baseColorFactor = originalPBRMetallicRoughness.baseColorFactor;
-                newMaterial.pbrMetallicRoughness.metallicFactor = originalPBRMetallicRoughness.metallicFactor;
-                newMaterial.pbrMetallicRoughness.roughnessFactor = originalPBRMetallicRoughness.roughnessFactor;
-            }
-        }
-        return newMaterial;
+/**
+ * Sets the glTF alpha mode to a glTF material from the Babylon Material
+ * @param glTFMaterial glTF material
+ * @param babylonMaterial Babylon material
+ */
+function setAlphaMode(glTFMaterial: IMaterial, babylonMaterial: Material & { alphaCutOff?: number }): void {

```

Comment:
Should non-exported functions at the module level also be PascalCase?

Reply:
Yes